### ISSUE TITLE
Keyboard focus partially obscured on sub-menu items

### ISSUE THREAD

**Issue by jonels-msft** [readthedocs/sphinx_rtd_theme#960]
### Problem

The keyboard focus rectangle on table of contents sub-menu items is partially covered and hard to see.

### Reproducible Project

https://docs.citusdata.com/en/v9.3/installation/multi_machine_rhel.html

#### Error Logs/Results

Here is a video demonstrating the problem: https://www.youtube.com/watch?v=LcBkzHHr30E

#### Expected Results

The entire focus rectangle should be visible, as specified by [WCAG 2.4.7](https://www.w3.org/WAI/WCAG21/Understanding/focus-visible.html)

### Environment Info

- Python Version: 3.7.7
- Sphinx Version: 1.4.9
- RTD Theme Version: 0.2.4
